### PR TITLE
Fix minor issues in the docs.

### PR DESCRIPTION
**Pull request by trilleplay**
Fixing some punctions and grammatical issues.

### CODE REVIEW

File: docs/Guides/archiving.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 #**What is archiving?**
-Archiving gives you the ability to collect messages by a specific user or in a channel and keep them in a text file format. By default this is available to moderators and above (permissions lvl 2 and higher). 
+Archiving gives you the ability to collect messages by a specific user or in a channel and keep them in a text file format. By default this is available to moderators and above (permissions lvl 2 and higher.)
```
Comment (AEnterprise) [inline docs/Guides/archiving.md]: i'm no lang expert but this looks wrong?, same for the other places you did this

Comment (Aimarekin) [reply docs/Guides/archiving.md]: @AEnterprise 

Yup, that point should be out of the parenthesis, or at least one inside and another outside, but that is often not done.

With the correction, you are declaring the end of the sentence inside of the parenthesis but you are declaring that the sentence that includes the parenthesis hasn't finished yet.

In short you *must* have a point after the parenthesis to declare the end of the sentence that contains the parenthesis, but It's not necessary to put a point to declare the end of the sentence inside the parenthesis, as the parenthesis already does that.

Comment (Aimarekin) [reply docs/Guides/archiving.md]: Same happens in https://github.com/AEnterprise/GearBot/pull/70/commits/1dda9c0cb35977f9f320cabbe742ba2d9f96477e , the point inside the parenthesis is not needed. However, you *must* put a point after the parenthesis there. The sentence outside the parenthesis is wrong.

Comment (trilleplay) [reply docs/Guides/archiving.md]: Oh welp, I've always done sentences like that, I'll correct the mess I've made in a sec, I'll leave the missing punctions and the missing capital I's though.

Comment (Aimarekin) [reply docs/Guides/archiving.md]: 👍

It's nice that you correct some issues tho :P

**Comment by trilleplay** [comment]
Everything should be fixed that is mentioned in the review now.

**Pull request merged**